onboard-rest-api/src/main/java/no/vegvesen/ixn/serviceprovider/model/PrivateChannelApi.java

```diff
@@ -0,0 +1,46 @@
+package no.vegvesen.ixn.serviceprovider.model;
+
+public class PrivateChannelApi {
+
+    private Integer id;
+    private String clientName;
```

Comment:
Just nitpicking, but don't you think it would be better to call it something along the lines of channelOwner, etc? Client is a bit overused, and the member variable actually refers to a Service Provider, not a client.